src/data/find-wallet/real-wallet-data.ts

```diff
@@ -0,0 +1,567 @@
+const walletData = [
```

Comment:
What is this "real" wallet data in `real-wallet-data.ts` vs `wallet-data.ts`. Is one just dummy data? We can just make 1 wallet data file, correct?

Reply:
Yup, we can swap these out now! Just didn't want to remove the dummy data when we started this on Friday whilst people were reviewing it still/potentially breaking changes to JSON structure.

Reply:
Will swap out today!